components/web-components/toggle/index.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+
+import * as S from './style'
+
+export type Size = 'sm' | 'lg'
+export type Brand = 'vpn' | 'shields'
+interface Props {
+  size?: Size
+  brand?: Brand
+  isOn?: boolean
+  disabled?: boolean
+  accessibleLabel?: string
+  onChange?: (isOn?: boolean) => unknown
+}
+
+function Toggle (props: Props) {
+  const [isOn, setIsOn] = React.useState(props.isOn ?? false)
+
+  const onToggleClick = () => {
+    const activated = !isOn
+    setIsOn(activated)
+    props.onChange?.(activated)
+  }
+
+  return (
+    <S.ToggleBox
+      type='button'
+      role='switch'
+      aria-checked={isOn}
+      aria-label={props.accessibleLabel}
+      onClick={onToggleClick}
+      isActive={isOn}
```

Comment:
I'm wondering what the reason to have 2 concepts of 'checked' - `isOn` and `isActive`? Should we stick with one of them? To me, 'active' might conflict with the ':active' css pseudo class, and I don't think it's being used to represent that here.